pkg/util/proxy.go

```diff
@@ -97,12 +97,15 @@ func (p *ProxyConfig) ProxyConfig() []string {
 
 	if p.httpProxy != "" {
 		config = append(config, fmt.Sprintf("HTTP_PROXY=%s", p.httpProxy))
+		config = append(config, fmt.Sprintf("http_proxy=%s", p.httpProxy))
 	}
 
 	if p.httpsProxy != "" {
 		config = append(config, fmt.Sprintf("HTTPS_PROXY=%s", p.httpsProxy))
+		config = append(config, fmt.Sprintf("https_proxy=%s", p.httpsProxy))
 	}
 	config = append(config, fmt.Sprintf("NO_PROXY=%s", p.NoProxy()))
+	config = append(config, fmt.Sprintf("no_proxy=%s", p.NoProxy()))
```

Comment:
@praveenkumar I was expecting that we would also change `ApplyToEnvironment()` function, but we are not.  We should also fix it, isn't it?

```
func (p *ProxyConfig) ApplyToEnvironment() {
	if !p.IsEnabled() {
		return
	}

	if p.httpProxy != "" {
		os.Setenv("HTTP_PROXY", p.httpProxy)
	}
	if p.httpsProxy != "" {
		os.Setenv("HTTPS_PROXY", p.httpsProxy)
	}
	os.Setenv("NO_PROXY", p.NoProxy())
}
```